Pilot linter errors

```
I1103 22:41:44.803] pilot/test/integration/driver.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:44.803] pilot/test/integration/infra.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:53.632] pilot/test/integration/driver.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.633] pilot/test/integration/infra.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.724] pilot/adapter/serviceregistry/aggregate/controller_test.go:80:40:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.725] pilot/adapter/serviceregistry/aggregate/controller_test.go:151:42:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.725] pilot/adapter/serviceregistry/aggregate/controller_test.go:212:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.726] pilot/adapter/serviceregistry/aggregate/controller_test.go:284:41:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.726] pilot/cmd/istioctl/main.go:547:41:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.772] pilot/model/validation.go:294:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.773] pilot/model/validation.go:450:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.773] pilot/model/validation.go:521:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.774] pilot/model/validation.go:545:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.774] pilot/model/validation.go:664:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.775] pilot/model/validation.go:717:46:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.775] pilot/model/validation.go:758:45:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.776] pilot/model/validation.go:893:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.776] pilot/model/validation.go:968:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.777] pilot/model/validation.go:999:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.777] pilot/model/validation.go:1003:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.777] pilot/model/validation.go:1007:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.778] pilot/model/validation.go:1013:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.778] pilot/model/validation.go:1018:15:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.778] pilot/platform/kube/controller.go:120:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.779] pilot/platform/kube/admit/admit_test.go:270:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.787] pilot/proxy/envoy/config.go:310:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.787] pilot/proxy/envoy/discovery_test.go:136:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.788] pilot/proxy/envoy/discovery_test.go:147:44:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.788] pilot/proxy/envoy/discovery_test.go:159:44:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.788] pilot/proxy/envoy/discovery_test.go:200:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.788] pilot/proxy/envoy/discovery_test.go:211:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.788] pilot/proxy/envoy/discovery_test.go:284:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.789] pilot/proxy/envoy/discovery_test.go:295:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.789] pilot/proxy/envoy/discovery_test.go:328:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.789] pilot/proxy/envoy/discovery_test.go:339:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.789] pilot/proxy/envoy/discovery_test.go:440:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.789] pilot/proxy/envoy/discovery_test.go:474:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.790] pilot/proxy/envoy/discovery_test.go:607:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.790] pilot/proxy/envoy/discovery_test.go:618:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.790] pilot/proxy/envoy/discovery_test.go:645:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.790] pilot/proxy/envoy/discovery_test.go:656:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.790] pilot/proxy/envoy/discovery_test.go:667:48:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.791] pilot/proxy/envoy/discovery_test.go:694:43:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.791] pilot/test/integration/egress_rules.go:125:24:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
```

--- a/pilot/adapter/serviceregistry/aggregate/controller_test.go
+++ b/pilot/adapter/serviceregistry/aggregate/controller_test.go
@@ -77,7 +77,7 @@ func buildMockController() *Controller {
 func TestServicesError(t *testing.T) {
 	aggregateCtl := buildMockController()
 
-	discovery1.ServicesError = errors.New("Mock Services() error")
+	discovery1.ServicesError = errors.New("mock Services() error")
 
 	// List Services from aggregate controller
 	_, err := aggregateCtl.Services()
@@ -148,7 +148,7 @@ func TestGetService(t *testing.T) {
 func TestGetServiceError(t *testing.T) {
 	aggregateCtl := buildMockController()
 
-	discovery1.GetServiceError = errors.New("Mock GetService() error")
+	discovery1.GetServiceError = errors.New("mock GetService() error")
 
 	// Get service from client with error
 	svc, err := aggregateCtl.GetService(mock.HelloService.Hostname)
@@ -209,7 +209,7 @@ func TestHostInstances(t *testing.T) {
 func TestHostInstancesError(t *testing.T) {
 	aggregateCtl := buildMockController()
 
-	discovery1.HostInstancesError = errors.New("Mock HostInstances() error")
+	discovery1.HostInstancesError = errors.New("mock HostInstances() error")
 
 	// Get Instances from client with error
 	instances, err := aggregateCtl.HostInstances(map[string]bool{mock.HelloInstanceV0: true})
@@ -281,7 +281,7 @@ func TestInstances(t *testing.T) {
 func TestInstancesError(t *testing.T) {
 	aggregateCtl := buildMockController()
 
-	discovery1.InstancesError = errors.New("Mock Instances() error")
+	discovery1.InstancesError = errors.New("mock Instances() error")
 
 	// Get Instances from client with error
 	instances, err := aggregateCtl.Instances(mock.HelloService.Hostname,

--- a/pilot/cmd/istioctl/main.go
+++ b/pilot/cmd/istioctl/main.go
@@ -544,7 +544,7 @@ func schema(configClient *crd.Client, typ string) (model.ProtoSchema, error) {
 			return desc, nil
 		}
 	}
-	return model.ProtoSchema{}, fmt.Errorf("Istio doesn't have configuration type %s, the types are %v",
+	return model.ProtoSchema{}, fmt.Errorf("configuration type %s not found, the types are %v",
 		typ, strings.Join(supportedTypes(configClient), ", "))
 }
 

--- a/pilot/model/validation.go
+++ b/pilot/model/validation.go
@@ -291,7 +291,8 @@ func ValidateMatchCondition(mc *proxyconfig.MatchCondition) (errs error) {
 		if err := ValidateL4MatchAttributes(mc.Udp); err != nil {
 			errs = multierror.Append(errs, err)
 		}
-		errs = multierror.Append(errs, fmt.Errorf("Istio does not support UDP protocol yet"))
+		// nolint: golint
+		errs = multierror.Append(errs, fmt.Errorf("UDP protocol not supported yet"))
 	}
 
 	if mc.Request != nil {
@@ -447,7 +448,7 @@ func ValidateL4Fault(fault *proxyconfig.L4FaultInjection) (errs error) {
 		if err := ValidateTerminate(fault.GetTerminate()); err != nil {
 			errs = multierror.Append(errs, err)
 		}
-		errs = multierror.Append(errs, fmt.Errorf("Istio does not support the terminate fault yet"))
+		errs = multierror.Append(errs, fmt.Errorf("the terminate fault not supported yet"))
 	}
 
 	if fault.GetThrottle() != nil {
@@ -518,7 +519,7 @@ func ValidateDelay(delay *proxyconfig.HTTPFaultInjection_Delay) (errs error) {
 		if err := ValidateDuration(delay.GetExponentialDelay()); err != nil {
 			errs = multierror.Append(errs, multierror.Prefix(err, "exponentialDelay invalid: "))
 		}
-		errs = multierror.Append(errs, fmt.Errorf("Istio does not support exponentialDelay yet"))
+		errs = multierror.Append(errs, fmt.Errorf("exponentialDelay not supported yet"))
 	}
 
 	return
@@ -542,7 +543,7 @@ func ValidateAbort(abort *proxyconfig.HTTPFaultInjection_Abort) (errs error) {
 	switch abort.ErrorType.(type) {
 	case *proxyconfig.HTTPFaultInjection_Abort_GrpcStatus:
 		// TODO No validation yet for grpc_status / http2_error / http_status
-		errs = multierror.Append(errs, fmt.Errorf("Istio does not support gRPC fault injection yet"))
+		errs = multierror.Append(errs, fmt.Errorf("gRPC fault injection not supported yet"))
 	case *proxyconfig.HTTPFaultInjection_Abort_Http2Error:
 		// TODO No validation yet for grpc_status / http2_error / http_status
 	case *proxyconfig.HTTPFaultInjection_Abort_HttpStatus:
@@ -661,7 +662,7 @@ func ValidateWeights(routes []*proxyconfig.DestinationWeight) (errs error) {
 
 	if sum != 100 {
 		errs = multierror.Append(errs,
-			fmt.Errorf("Route weights total %v (must total 100)", sum))
+			fmt.Errorf("route weights total %v (must total 100)", sum))
 	}
 
 	return
@@ -714,6 +715,7 @@ func ValidateRouteRule(msg proto.Message) error {
 		}
 
 		if value.WebsocketUpgrade {
+			// nolint: golint
 			errs = multierror.Append(errs, errors.New("WebSocket upgrade is not allowed on redirect rules"))
 		}
 	}
@@ -755,6 +757,7 @@ func ValidateRouteRule(msg proto.Message) error {
 		if err := ValidateL4Fault(value.L4Fault); err != nil {
 			errs = multierror.Append(errs, err)
 		}
+		// nolint: golint
 		errs = multierror.Append(errs, fmt.Errorf("L4 faults are not implemented"))
 	}
 
@@ -890,7 +893,7 @@ func ValidateEgressRulePort(port *proxyconfig.EgressRule_Port) error {
 	switch protocol {
 	case ProtocolHTTP, ProtocolHTTPS, ProtocolHTTP2, ProtocolGRPC:
 	default:
-		return fmt.Errorf("Support for non-HTTP protocols is not yet available")
+		return fmt.Errorf("support for non-HTTP protocols is not yet available")
 	}
 
 	return nil
@@ -965,7 +968,7 @@ func ValidateDuration(pd *duration.Duration) error {
 		return errors.New("duration must be greater than 1ms")
 	}
 	if dur%time.Millisecond != 0 {
-		return errors.New("Istio only supports durations to ms precision")
+		return errors.New("only durations to ms precision is supported")
 	}
 	return nil
 }
@@ -996,26 +999,26 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs 
 
 	if drainDuration%time.Second != 0 {
 		errs = multierror.Append(errs,
-			errors.New("Istio drain time only supports durations to seconds precision"))
+			errors.New("drain time only supports durations to seconds precision"))
 	}
 	if parentShutdownDuration%time.Second != 0 {
 		errs = multierror.Append(errs,
-			errors.New("Istio parent shutdown time only supports durations to seconds precision"))
+			errors.New("parent shutdown time only supports durations to seconds precision"))
 	}
 	if parentShutdownDuration <= drainDuration {
 		errs = multierror.Append(errs,
-			fmt.Errorf("Istio parent shutdown time %v must be greater than drain time %v",
+			fmt.Errorf("parent shutdown time %v must be greater than drain time %v",
 				parentShutdownDuration.String(), drainDuration.String()))
 	}
 
 	if drainDuration > drainTimeMax {
 		errs = multierror.Append(errs,
-			fmt.Errorf("Istio drain time %v must be <%v", drainDuration.String(), drainTimeMax.String()))
+			fmt.Errorf("drain time %v must be <%v", drainDuration.String(), drainTimeMax.String()))
 	}
 
 	if parentShutdownDuration > parentShutdownTimeMax {
 		errs = multierror.Append(errs,
-			fmt.Errorf("Istio parent shutdown time %v must be <%v",
+			fmt.Errorf("parent shutdown time %v must be <%v",
 				parentShutdownDuration.String(), parentShutdownTimeMax.String()))
 	}
 

--- a/pilot/platform/kube/admit/admit_test.go
+++ b/pilot/platform/kube/admit/admit_test.go
@@ -267,7 +267,7 @@ func makeTestData(t *testing.T, valid bool) []byte {
 func makeTestClient() (*http.Client, error) {
 	clientCert, err := tls.X509KeyPair(testcerts.ClientCert, testcerts.ClientKey)
 	if err != nil {
-		return nil, fmt.Errorf("LoadX509KeyPair failed: %v", err)
+		return nil, fmt.Errorf("failed to load X509KeyPair: %v", err)
 	}
 	caCertPool := x509.NewCertPool()
 	caCertPool.AppendCertsFromPEM(testcerts.CACert)

--- a/pilot/platform/kube/controller.go
+++ b/pilot/platform/kube/controller.go
@@ -117,7 +117,7 @@ func NewController(client kubernetes.Interface, options ControllerOptions) *Cont
 // Returning an error causes repeated execution of the entire chain.
 func (c *Controller) notify(obj interface{}, event model.Event) error {
 	if !c.HasSynced() {
-		return errors.New("Waiting till full synchronization")
+		return errors.New("waiting till full synchronization")
 	}
 	k, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {

--- a/pilot/proxy/envoy/config.go
+++ b/pilot/proxy/envoy/config.go
@@ -307,7 +307,7 @@ func buildRDSRoute(mesh *proxyconfig.MeshConfig, node proxy.Node, routeName stri
 		httpConfigs = buildOutboundHTTPRoutes(mesh, node, instances, services, config)
 		httpConfigs = buildEgressHTTPRoutes(mesh, node, instances, config, httpConfigs)
 	default:
-		return nil, errors.New("Unrecognized node type")
+		return nil, errors.New("unrecognized node type")
 	}
 
 	if routeName == RDSAll {

--- a/pilot/proxy/envoy/discovery_test.go
+++ b/pilot/proxy/envoy/discovery_test.go
@@ -133,7 +133,7 @@ func TestServiceDiscoveryListAllServices(t *testing.T) {
 
 func TestServiceDiscoveryListAllServicesError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := "/v1/registration/"
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -144,7 +144,7 @@ func TestServiceDiscoveryListAllServicesError(t *testing.T) {
 
 func TestServiceDiscoveryListAllServicesError2(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.InstancesError = errors.New("Mock Instances() error")
+	mockDiscovery.InstancesError = errors.New("mock Instances() error")
 	url := "/v1/registration/" + mock.HelloService.Key(mock.HelloService.Ports[0],
 		map[string]string{"version": "v1"})
 	response := getDiscoveryResponse(ds, "GET", url, t)
@@ -156,7 +156,7 @@ func TestServiceDiscoveryListAllServicesError2(t *testing.T) {
 
 func TestServiceDiscoveryError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.InstancesError = errors.New("Mock Instances() error")
+	mockDiscovery.InstancesError = errors.New("mock Instances() error")
 	url := "/v1/registration/nonexistent"
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -197,7 +197,7 @@ func TestClusterDiscovery(t *testing.T) {
 
 func TestClusterDiscoveryError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/clusters/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -208,7 +208,7 @@ func TestClusterDiscoveryError(t *testing.T) {
 
 func TestClusterDiscoveryError2(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/clusters/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -281,7 +281,7 @@ func TestClusterDiscoveryIngress(t *testing.T) {
 
 func TestClusterDiscoveryIngressError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/clusters/%s/%s", "istio-proxy", mock.Ingress.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -292,7 +292,7 @@ func TestClusterDiscoveryIngressError(t *testing.T) {
 
 func TestClusterDiscoveryRouterError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/clusters/%s/%s", "istio-proxy", mock.Router.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -325,7 +325,7 @@ func TestRouteDiscoveryV0(t *testing.T) {
 
 func TestRouteDiscoveryError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/routes/80/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -336,7 +336,7 @@ func TestRouteDiscoveryError(t *testing.T) {
 
 func TestRouteDiscoveryError2(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/routes/80/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -437,7 +437,7 @@ func TestRouteDiscoveryWebsocket(t *testing.T) {
 
 func TestRouteDiscoveryIngressError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/routes/80/%s/%s", "istio-proxy", mock.Ingress.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -471,7 +471,7 @@ func TestRouteDiscoveryIngressWeighted(t *testing.T) {
 
 func TestRouteDiscoveryRouterError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/routes/80/%s/%s", "istio-proxy", mock.Router.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -604,7 +604,7 @@ func TestListenerDiscoverySidecarAuthOptOut(t *testing.T) {
 
 func TestRouteDiscoverySidecarError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.HelloProxyV1.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -615,7 +615,7 @@ func TestRouteDiscoverySidecarError(t *testing.T) {
 
 func TestRouteDiscoverySidecarError2(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.HelloProxyV1.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -642,7 +642,7 @@ func TestListenerDiscoveryIngress(t *testing.T) {
 
 func TestListenerDiscoverySidecarError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -653,7 +653,7 @@ func TestListenerDiscoverySidecarError(t *testing.T) {
 
 func TestListenerDiscoverySidecarError2(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.HelloProxyV0.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -664,7 +664,7 @@ func TestListenerDiscoverySidecarError2(t *testing.T) {
 
 func TestListenerDiscoveryIngressError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.HostInstancesError = errors.New("Mock HostInstances() error")
+	mockDiscovery.HostInstancesError = errors.New("mock HostInstances() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.Ingress.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {
@@ -691,7 +691,7 @@ func TestListenerDiscoveryHttpProxy(t *testing.T) {
 
 func TestListenerDiscoveryRouterError(t *testing.T) {
 	_, _, ds := commonSetup(t)
-	mockDiscovery.ServicesError = errors.New("Mock Services() error")
+	mockDiscovery.ServicesError = errors.New("mock Services() error")
 	url := fmt.Sprintf("/v1/listeners/%s/%s", "istio-proxy", mock.Router.ServiceNode())
 	response := getDiscoveryResponse(ds, "GET", url, t)
 	if response.StatusCode != http.StatusServiceUnavailable {

--- a/pilot/proxy/envoy/ingress_test.go
+++ b/pilot/proxy/envoy/ingress_test.go
@@ -1,3 +1,17 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package envoy
 
 import (

--- a/pilot/proxy/resolve.go
+++ b/pilot/proxy/resolve.go
@@ -1,3 +1,17 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package proxy
 
 import (

--- a/pilot/test/integration/driver.go
+++ b/pilot/test/integration/driver.go
@@ -46,8 +46,8 @@ var (
 
 	// Enable/disable auth, or run both for the tests.
 	authmode string
-	verbose   bool
-	count     int
+	verbose  bool
+	count    int
 
 	// The particular test to run, e.g. "HTTP reachability" or "routing rules"
 	testType string

--- a/pilot/test/integration/egress_rules.go
+++ b/pilot/test/integration/egress_rules.go
@@ -122,7 +122,7 @@ func (t *egressRules) verifyReachable(url string, shouldBeReachable bool) error 
 				resp := t.clientRequest(src, url, 1, fmt.Sprintf("-key Trace-Id -val %q", trace))
 				reachable := len(resp.code) > 0 && resp.code[0] == httpOk && strings.Contains(resp.body, trace)
 				if reachable && !shouldBeReachable {
-					return fmt.Errorf("Error: %s is reachable from %s (should be unreachable)", url, src)
+					return fmt.Errorf("%s is reachable from %s (should be unreachable)", url, src)
 				}
 				if !reachable && shouldBeReachable {
 					return errAgain

--- a/pilot/test/integration/infra.go
+++ b/pilot/test/integration/infra.go
@@ -72,7 +72,7 @@ type infra struct { // nolint: aligncheck
 
 	// store error logs in specific directory
 	errorLogsDir string
-	
+
 	namespaceCreated      bool
 	istioNamespaceCreated bool
 	debugImagesAndMode    bool

